dask_cuda/device_host_file.py

```diff
@@ -186,3 +189,15 @@ def __iter__(self):
     def __delitem__(self, key):
         self.device_keys.discard(key)
         del self.device_buffer[key]
+
+    def get_from_dev_or_host_buffer(self, key):
+        """Get stored value without moving it from host to device memory.
+        If the value isn't in device memory, `DeviceSerialized` is returned.
+        """
+        if key not in self.device_keys or key in self.device_func:
+            return self.__getitem__(key)
+        else:
+            return self.host_buffer[key]
```

Comment:
Not sure if I'm missing some condition here, but wouldn't it be simpler to write:
```python
        if key in self.host_buffer:
            return self.host_buffer[key]
        else:
            return self.__getitem__(key)
```